planner/core/stringer.go

```diff
@@ -226,3 +226,144 @@ func toString(in Plan, strs []string, idxs []int) ([]string, []int) {
 	strs = append(strs, str)
 	return strs, idxs
 }
+
+// ToStringWithCount explains a Plan, returns description string.
+func ToStringWithCount(p Plan) string {
+	strs, _ := toStringWithCount(p, []string{}, []int{})
+	return strings.Join(strs, "->")
+}
+
+func toStringWithCount(in Plan, strs []string, idxs []int) ([]string, []int) {
+	switch x := in.(type) {
+	case LogicalPlan:
+		if len(x.Children()) > 1 {
+			idxs = append(idxs, len(strs))
+		}
+
+		for _, c := range x.Children() {
+			strs, idxs = toString(c, strs, idxs)
+		}
+	case PhysicalPlan:
+		if len(x.Children()) > 1 {
+			idxs = append(idxs, len(strs))
+		}
+
+		for _, c := range x.Children() {
+			strs, idxs = toString(c, strs, idxs)
+		}
+	}
+
+	var str string
+	switch x := in.(type) {
+	case *CheckTable:
+		str = "CheckTable"
+	case *PhysicalIndexScan:
+		str = fmt.Sprintf("Index(%s.%s) %.2f", x.Table.Name.L, x.Index.Name.L, x.StatsCount())
```

Comment:
Range is not needed?

Reply:
No, it may be too long and can be inferred from the sql.